### PR TITLE
Fix npm distribution

### LINKED ISSUE #163: Including smooch via NPM fails due to missing /dist directory
Hi Marc-Antoine Lemieux,

I'm trying to include smooch using browserify. I'm getting the following when executing task:

```
Running "browserify:vendor" (browserify) task
>> Error: Cannot find module 'smooch' from '/vagrant/sails/portal/assets/js'
Warning: Error running grunt-browserify. Use --force to continue.
```

I've taken latest from NPM (2.1.1), packages.config defines main as `dist/smooch.js` which doesn't exist.

### PR DESCRIPTION
**Pull request by lemieux**
This should fix #163 . The problem was that npm is using the .gitignore file if no .npmignore file is provided and was ignoring the dist folder.

I tested it against one of my projects and it worked.

@alavers @mspensieri @dannytranlx

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -0,0 +1,28 @@
+dist/stats.json
```
Comment (mspensieri) [inline .npmignore]: Would it be easier to just ignore everything, and add reverse rules for the stuff you want to keep?

File: .npmignore [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+webpack/
+Gruntfile.js
+karma.conf.js
+.babelrc
+.csscomb.json
+.esformatter
+.eslintignore
+.eslintrc
+.jsfmtrc
+bower.json
```
Comment (dannytranlx) [inline .npmignore]: Add `.jshintrc`?

Comment (dannytranlx) [inline .npmignore]: And `smooch-js.sublime-project`

File: .npmignore (hunk 1 above)
Comment (dannytranlx) [inline .npmignore]: I think @jugarrit tried something like that before without success :confused:

**Comment by alavers** [comment]
:+1:

File: .npmignore (hunk 1 above)
Comment (lemieux) [inline .npmignore]: also, this doesn't have a big impact in the end, it's just to be nice and put only what is needed in the npm package... if we forget to ignore stuff later, it won't break anything as long as `dist/smooch.js` is still there.

File: .npmignore (hunk 2 above)
Comment (lemieux) [inline .npmignore]: sublime project is covered by `*.sublime-project`, I'll add `jshinrc`

Comment (lemieux) [inline .npmignore]: fixed

**Comment by dannytranlx** [comment]
:+1:

**Comment by mspensieri** [comment]
:+1:

**Pull request merged**